packages/@ama-sdk/generator-sdk/src/generators/shell/templates/base/package.json

```diff
@@ -123,4 +123,4 @@
     "npm": ">=6.0.0"
   },
   "packageManager": "yarn@3.4.1"
-}
+}
```

Comment:
missing empty line at the eof

Reply:
Oops I didn't see the comment before merging.